File: support_tools/swigwin-3.0.12/Examples/test-suite/go/rename_scope_runme.go
package main

import . "./rename_scope"

func main() {
	a := NewNatural_UP()
	b := NewNatural_BP()

	if a.Rtest() != 1 {
		panic(0)
	}

	if b.Rtest() != 1 {
		panic(0)
	}

	_ = Equals
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/javascript/rename_scope_runme.js
var rename_scope = require("rename_scope");

var a = new rename_scope.Natural_UP();
var b = new rename_scope.Natural_BP();

if (a.rtest() !== 1) {
  throw new Error("a.rtest(): Expected 1, was " + a.rtest());
}

if (b.rtest() !== 1) {
  throw new Error("b.rtest(): Expected 1, was " + b.rtest());
}

var f = rename_scope.equals;
if (f === undefined) {
  throw new Error("Equality operator has not been renamed.");
}
